python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -160,9 +161,56 @@ def _get_s3_storage_config(cfg):
     assert not s3_storage.https
     assert s3_storage.region == ""
     assert not s3_storage.use_virtual_addressing
+    assert s3_storage.ca_cert_path == ""
+    assert s3_storage.ca_cert_dir == ""
 
     assert "sym" in ac["test"].list_symbols()
 
+class ParameterDisplayStatus(Enum):
+    NOT_SHOW = 1
```

Comment:
`NOT_SHOW` is unused?

I think we can just replace this enum with a boolean True/False

Reply:
It is not unused. The `if` case below is `if` and `elif`